Makefile

```diff
@@ -120,6 +122,13 @@ _verify_scss_lint:
 # HACK: Use backticks rather than xargs because xargs swallow exit codes (everything becomes 1 and stoopidly scss-lint exits with 1 if warnings, 2 if errors)
 	@if [ -e .scss-lint.yml ]; then { scss-lint -c ./.scss-lint.yml `$(call GLOB,'*.scss')`; if [ $$? -ne 0 -a $$? -ne 1 ]; then exit 1; fi; $(DONE); } fi
 
+_verify_pa11y:
```

Comment:
@TheoLeanse @GlynnPhillips it's this bit I'm not sure about

I want pa11y to be part of verify. If you do `make verify` locally, it would run pa11y locally (you'd need to have the app running first), but when verify is run in CI I'd want it to run pa11y on the deployed app. Possible?